### PR TITLE
Reduce filesizes for mint tests

### PR DESCRIPTION
**Pull request by poornas**

### CODE REVIEW

**Comment by harshavardhana** [comment]
thank you for this @poornas - geat help.

File: run/core/worm/quick-worm-tests.go [hunk 1]
```diff
@@ -248,7 +248,7 @@ func testPutMultipart(s3Client *s3.S3) {
 		"expiry":     expiry,
 	}
 	function := "PutMultiPart"
-	file, err := os.Open("/mint/data/datafile-65-MB")
+	file, err := os.Open("/mint/data/datafile-5-MB")
```
Comment (kannappanr) [inline run/core/worm/quick-worm-tests.go]: This probably has to be a number greater than 5 MB for multi-part testing to happen

File: build/aws-sdk-java/src/FunctionalTests.java [hunk 2]
```diff
@@ -530,25 +524,26 @@ public static void downloadGetRangeEncryption_test6() throws Exception {
         long startTime = System.currentTimeMillis();
 
         String objectName = "testobject";
-        int start = 64*1024;
-        int length = 1024*1024-64*1024;
-        String range1MD5 = Utils.getFileMD5(file1Mb, start, length);
+        int start = 64 * 1024;
```
Comment (kannappanr) [inline build/aws-sdk-java/src/FunctionalTests.java]: wrong parameters to test `Range`.

File: build/aws-sdk-java/src/FunctionalTests.java [hunk 3]
```diff
@@ -494,33 +487,34 @@ public static void downloadGetRangeEncryption_test5() throws Exception {
         long startTime = System.currentTimeMillis();
 
         String objectName = "testobject";
-        int start = 64*1024;
-        int length = 64*1024;
-        String range1MD5 = Utils.getFileMD5(file1Mb, start, length);
+        int start = 64 * 1024;
```
Comment (kannappanr) [inline build/aws-sdk-java/src/FunctionalTests.java]: please fix `start` and `length` values

File: build/aws-sdk-java/src/FunctionalTests.java [hunk 4]
```diff
@@ -459,32 +452,32 @@ public static void downloadGetRangeEncryption_test4() throws Exception {
 
         String objectName = "testobject";
         int start = 1;
-        int length = 1024*1024-1;
-        String range1MD5 = Utils.getFileMD5(file1Mb, start, length);
+        int length = 1024 * 1024 - 1;
```
Comment (kannappanr) [inline build/aws-sdk-java/src/FunctionalTests.java]: `length=1024-1`

File: build/aws-sdk-java/src/FunctionalTests.java [hunk 5]
```diff
@@ -423,32 +416,32 @@ public static void downloadGetRangeEncryption_test3() throws Exception {
 
         String objectName = "testobject";
         int start = 0;
-        int length = 1024*1024-1;
-        String range1MD5 = Utils.getFileMD5(file1Mb, start, length);
+        int length = 1024 * 1024 - 1;
+        String range1MD5 = Utils.getFileMD5(file1Kb, start, length);
```
Comment (kannappanr) [inline build/aws-sdk-java/src/FunctionalTests.java]: `length=1024-1`

File: build/aws-sdk-java/src/FunctionalTests.java [hunk 6]
```diff
@@ -349,33 +343,33 @@ public static void downloadGetRangeEncryption_test1() throws Exception {
         long startTime = System.currentTimeMillis();
 
         String objectName = "testobject";
-        String range1MD5 = Utils.getFileMD5(file1Mb);
+        String range1MD5 = Utils.getFileMD5(file1Kb);
         int start = 0;
-        int length = 1024*1024;
+        int length = 1024 * 1024;
```
Comment (kannappanr) [inline build/aws-sdk-java/src/FunctionalTests.java]: `length=1024`

File: run/core/worm/quick-worm-tests.go (hunk 1 above)
Comment (poornas) [reply run/core/worm/quick-worm-tests.go]: since this is explicitly uploaded with uploadPart and there is only one part, 5MB is good enough

Comment (kannappanr) [reply run/core/worm/quick-worm-tests.go]: ok

**Comment by kannappanr** [comment]
Please also check Travis failure

**Comment by nitisht** [comment]
@poornas I tried building locally, I see this - any ideas

```
[ivy:retrieve] 
[ivy:retrieve] :: problems summary ::
[ivy:retrieve] :::: WARNINGS
[ivy:retrieve] 	io problem while parsing ivy file: http://repo1.maven.org/maven2/commons-logging/commons-logging/1.1.3/commons-logging-1.1.3.pom: Impossible to load parent for file:/root/.ant/cache/commons-logging/commons-logging/ivy-1.1.3.xml.original. Parent=org.apache.commons#commons-parent;28
[ivy:retrieve] 		module not found: commons-logging#commons-logging;1.1.3
[ivy:retrieve] 	==== local: tried
[ivy:retrieve] 	  /root/.ant/local/org.apache.commons/commons-parent/28/ivys/ivy.xml
[ivy:retrieve] 	  -- artifact org.apache.commons#commons-parent;28!commons-parent.jar:
[ivy:retrieve] 	  /root/.ant/local/org.apache.commons/commons-parent/28/jars/commons-parent.jar
[ivy:retrieve] 	==== shared: tried
[ivy:retrieve] 	  /root/.ant/shared/org.apache.commons/commons-parent/28/ivys/ivy.xml
[ivy:retrieve] 	  -- artifact org.apache.commons#commons-parent;28!commons-parent.jar:
[ivy:retrieve] 	  /root/.ant/shared/org.apache.commons/commons-parent/28/jars/commons-parent.jar
[ivy:retrieve] 	==== public: tried
[ivy:retrieve] 	  http://repo1.maven.org/maven2/org/apache/commons/commons-parent/28/commons-parent-28.pom
[ivy:retrieve] 	  -- artifact org.apache.commons#commons-parent;28!commons-parent.jar:
[ivy:retrieve] 	  http://repo1.maven.org/maven2/org/apache/commons/commons-parent/28/commons-parent-28.jar
[ivy:retrieve] 	io problem while parsing ivy file: http://repo1.maven.org/maven2/org/apache/httpcomponents/httpcomponents-client/4.5.2/httpcomponents-client-4.5.2.pom: Impossible to load parent for file:/root/.ant/cache/org.apache.httpcomponents/httpcomponents-client/ivy-4.5.2.xml.original. Parent=org.apache.httpcomponents#project;7
[ivy:retrieve] 	io problem while parsing ivy file: http://repo1.maven.org/maven2/org/apache/httpcomponents/httpclient/4.5.2/httpclient-4.5.2.pom: Impossible to load parent for file:/root/.ant/cache/org.apache.httpcomponents/httpclient/ivy-4.5.2.xml.original. Parent=org.apache.httpcomponents#httpcomponents-client;4.5.2
[ivy:retrieve] 		module not found: org.apache.httpcomponents#httpclient;4.5.2
[ivy:retrieve] 	==== local: tried
[ivy:retrieve] 	  /root/.ant/local/org.apache.httpcomponents/project/7/ivys/ivy.xml
[ivy:retrieve] 	  -- artifact org.apache.httpcomponents#project;7!project.jar:
[ivy:retrieve] 	  /root/.ant/local/org.apache.httpcomponents/project/7/jars/project.jar
[ivy:retrieve] 	==== shared: tried
[ivy:retrieve] 	  /root/.ant/shared/org.apache.httpcomponents/project/7/ivys/ivy.xml
[ivy:retrieve] 	  -- artifact org.apache.httpcomponents#project;7!project.jar:
[ivy:retrieve] 	  /root/.ant/shared/org.apache.httpcomponents/project/7/jars/project.jar
[ivy:retrieve] 	==== public: tried
[ivy:retrieve] 	  http://repo1.maven.org/maven2/org/apache/httpcomponents/project/7/project-7.pom
[ivy:retrieve] 	  -- artifact org.apache.httpcomponents#project;7!project.jar:
[ivy:retrieve] 	  http://repo1.maven.org/maven2/org/apache/httpcomponents/project/7/project-7.jar
[ivy:retrieve] 		::::::::::::::::::::::::::::::::::::::::::::::
[ivy:retrieve] 		::          UNRESOLVED DEPENDENCIES         ::
[ivy:retrieve] 		::::::::::::::::::::::::::::::::::::::::::::::
[ivy:retrieve] 		:: commons-logging#commons-logging;1.1.3: not found
[ivy:retrieve] 		:: org.apache.httpcomponents#httpclient;4.5.2: not found
[ivy:retrieve] 		::::::::::::::::::::::::::::::::::::::::::::::
[ivy:retrieve] 
[ivy:retrieve] :: USE VERBOSE OR DEBUG MESSAGE LEVEL FOR MORE DETAILS

BUILD FAILED
/mint/build/aws-sdk-java/build.xml:25: impossible to resolve dependencies:
	resolve failed - see output for details

Total time: 32 seconds
cp: cannot stat 'build/jar/FunctionalTests.jar': No such file or directory

```

**Comment by nitisht** [comment]
Looks like failure was a false alarm. Build works fine now

**Pull request merged**